pkgs/build-support/bintools-wrapper/add-flags.sh

```diff
@@ -25,6 +26,10 @@ if [ -e @out@/nix-support/libc-ldflags ]; then
     NIX_LDFLAGS_@suffixSalt@+=" $(< @out@/nix-support/libc-ldflags)"
 fi
 
+if [ -z "$NIX_DYNAMIC_LINKER_@suffixSalt@" ] && [ -e @out@/nix-support/dynamic-linker ]; then
```

Comment:
```suggestion
if [ -z "$NIX_DYNAMIC_LINKER_@suffixSalt@" && -e @out@/nix-support/dynamic-linker ]; then
```

Could we do this or does that not work with sh?

Reply:
That's not how shell works

Reply:
You mean sh? Because bash works like this.

Reply:
No?

```shell
$ echo $BASH_VERSION
4.4.23(1)-release
$ [ -z lal && -n lal ]
bash: [: missing `]'
```

Reply:
maybe needs `[[`...`]]`

Reply:
> maybe needs `[[`...`]]`

Yeah, that only work with new test. nvm.